pkg/agent/openflow/network_policy.go

```diff
@@ -1558,13 +1569,13 @@ func (c *client) UninstallPolicyRuleFlows(ruleID uint32) ([]string, error) {
 	}
 	staleOFPriorities := c.featureNetworkPolicy.getStalePriorities(conj)
 	// Delete action flows from the OVS bridge.
-	if err := c.ofEntryOperations.DeleteAll(conj.actionFlows); err != nil {
-		return nil, err
+	var flowMessages []*openflow15.FlowMod
+	for _, msg := range append(conj.actionFlows, conj.metricFlows...) {
+		flowMessages = append(flowMessages, msg)
 	}
-	if err := c.ofEntryOperations.DeleteAll(conj.metricFlows); err != nil {
+	if err := c.changeAll(map[ofAction][]*openflow15.FlowMod{del: flowMessages}); err != nil {
```

Comment:
The parameter of `DeleteAll` is interface "OFEntry". The struct `pkg/ovs/openflow.ofFlow` is the implementation of this interface. Since we change to cache the OpenFlow modification message in the struct, resources read from the cache are not DeleteAll's required type any more.

Reply:
updated.